packages/core/src/internal/sagaHelpers/throttle.js

```diff
@@ -1,34 +1,55 @@
+import * as is from '@redux-saga/is'
 import fsmIterator, { safeName } from './fsmIterator'
 import { take, fork, actionChannel, delay } from '../io'
 import * as buffers from '../buffers'
 
-export default function throttle(delayLength, pattern, worker, ...args) {
+export default function throttle(delayLength, patternOrChannel, worker, ...args) {
   let action, channel
 
-  const yActionChannel = { done: false, value: actionChannel(pattern, buffers.sliding(1)) }
   const yTake = () => ({ done: false, value: take(channel) })
-  const yFork = ac => ({ done: false, value: fork(worker, ...args, ac) })
+  const yFork = (ac) => ({ done: false, value: fork(worker, ...args, ac) })
```

Comment:
hm, locally i dont end up with those parens - are you sure that your IDE is using the local version of Prettier?

Reply:
You're right, I have my IDE setup to autoformat based on my preferences.